albatross/Common

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (C) 2019 Swift Navigation Inc.
+ * Contact: Swift Navigation <dev@swiftnav.com>
+ *
+ * This source is subject to the license found in the file 'LICENSE' which must
+ * be distributed together with this source. All other rights reserved.
+ *
+ * THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY OF ANY KIND,
+ * EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE IMPLIED
+ * WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A PARTICULAR PURPOSE.
+ */
+
+#ifndef ALBATROSS_COMMON_H
+#define ALBATROSS_COMMON_H
+
+#include <cereal/cereal.hpp>
+#include <cereal/archives/binary.hpp>
+#include <cereal/archives/json.hpp>
+#include <cereal/archives/xml.hpp>
+#include <cereal/types/polymorphic.hpp>
+
+#include <Eigen/Core>
+#include <Eigen/Dense>
+
+#include <math.h>
+#include <iomanip>
+#include <map>
+#include <string>
+#include <iostream>
+#include <unordered_map>
+#include <functional>
+#include <memory>
+#include <vector>
+#include <numeric>
+
+#include "map_utils.h"
+#include "random_utils.h"
+#include "cereal/eigen.h"
+
+#endif
```

Comment:
Incredibly minor nit: It's good to have a newline at the end of files. It's no longer required by the C++ standard (it was pre-C++11!), but git and other tools show a little indicator and really dumb text manipulating tools (sed/grep/etc) sometimes rely on the last line of a file ending with a newline.

Reply:
done